app/controllers/books_controller.rb

```diff
@@ -1,6 +1,18 @@
 class BooksController < ApplicationController
+  def index
+    if params[:search]
+      @books = BookFacade.find_book(params[:search])
+    # else
```

Comment:
if we're pushing this to main, do we want to leave it commented out ? or should we satisfy the sad path before we merge?

Reply:
merging for now as i am taking over where mal left off. I will delete the commented out code or note why it is there in the future PR